content/languages/c/authoring/memory-management-techniques.md

```diff
@@ -0,0 +1,24 @@
+---
+kind: tutorial
+languages: [c]
+sidebar: "language:c"
+---
+
+# Memory Management in C kata
+
+_TBD_
+
+## Arrays and strings
+
+- malloc in solution and free in tests
+- pass in a preallocated buffer (use size hints if possible)
+- two functions: get size, allocate in tests, run solution
+- two functions: solution with allocation, deallocation. Bookkeeping information managed by user or passed as additional `void*`
+- one function: accept buffer+size, return retsult or error and required size
```

Comment:
```suggestion
- one function: accept buffer+size, return result or error and required size
```

Reply:
This file is going to have a separate PR, there's no need to fix typos here.